modules/highgui/include/opencv2/highgui/highgui_c.h

```diff
@@ -365,7 +365,7 @@ enum
     CV_CAP_PROP_CONVERT_RGB   =16,
     CV_CAP_PROP_WHITE_BALANCE_U =17,
     CV_CAP_PROP_RECTIFICATION =18,
-    CV_CAP_PROP_MONOCROME     =19,
```

Comment:
`CV_CAP_PROP_MONOCROME = CV_CAP_PROP_MONOCHROME`

We should add this to comply with the remark of @vpisarev